src/State/HubInterface.php

```diff
@@ -119,6 +119,12 @@ public function captureLastError(): ?string;
      */
     public function addBreadcrumb(Breadcrumb $breadcrumb): bool;
 
+    /**
+     * Await all async transports, this effectively forces all async event to be
+     * transported immediately.
+     */
+    public function await(): void;
```

Comment:
This is a breaking change, if anyone extended the Hub.

Reply:
This is true, probably the best to keep it on our own hub implementation for now.